### PR TITLE
feat(navigation-page): add navigation page template - INNO-449

### PR DESCRIPTION
**Pull request by kalinchernev**
I'm not sure how it will best to handle the options in the select and if the flags are big enough. I tried to use the flags we have rather than external images for the demo.

### CODE REVIEW

File: framework/templates/navigation-page/navigation-page.twig [hunk 1]
```diff
@@ -1,2 +1,92 @@
-<h2>Wireframe</h2>
-<img src="{{ '/references/navigation-page/navigation-page--wireframe.png' | path }}" />
+{% include "@headers" %}
+
+{% embed "@sections--plain" with {'classes': 'ecl-pv-l ffa-bg-accent'} %}
+  {% block body %}
+    <div class="section__body">
+      <h2 class="ecl-h2 ecl-mb-l ecl-pa-none ffa-text-center ffa-text-white">
```
Comment (emeryro) [inline framework/templates/navigation-page/navigation-page.twig]: We may remove some of these classes, as next element in DOM is a heading (so no need to manually add a margin).
Something like "ecl-h2 ecl-pt-none ffa-text-center ffa-text-white" should be enough.

File: framework/templates/navigation-page/navigation-page.twig [hunk 2]
```diff
@@ -1,2 +1,92 @@
...
+        What we do by
+        <select name="topics">
+            <option value="regions-countries">Regions and countries</option>
+            <option value="themes">Themes</option>
+            <option value="partners">Partners</option>
+        </select>
+      </h2>
+
+      <h3 class="ecl-h2 ecl-mb-l ecl-pa-none ffa-text-white">Sahel & Iake Chad</h3>
```
Comment (emeryro) [inline framework/templates/navigation-page/navigation-page.twig]: Classes should be updated too.
I think that "ecl-h3 ffa-text-white" should be enough here.

File: framework/templates/navigation-page/navigation-page.twig [hunk 3]
```diff
@@ -1,2 +1,92 @@
...
+      <div class="ecl-row">
+        {% for country in countries %}
+          <div class="ecl-col ecl-col-sm-6 ecl-col-md-4 ecl-col-xl-3">
```
Comment (emeryro) [inline framework/templates/navigation-page/navigation-page.twig]: I will update cards component display in another ticket (INNO-461) to use image as background. So display of flags will be improved.
For now we can keep it as is; to be updated once my task is done.

File: framework/templates/navigation-page/navigation-page.twig [hunk 4]
```diff
@@ -1,2 +1,92 @@
...
+            {% include "@cards" with {
+              'children': country.children,
+              'label': country.label,
+              'link': '#',
+              'cardClass': 'ffa-block-center ecl-mb-m'
+            } %}
+          </div>
+        {% endfor %}
```
Comment (emeryro) [inline framework/templates/navigation-page/navigation-page.twig]: Maybe we can add another category (like "Sahel & Iake Chad"), to display how it looks like.

File: framework/templates/navigation-page/navigation-page.twig [hunk 5]
```diff
@@ -1,2 +1,92 @@
-<h2>Wireframe</h2>
-<img src="{{ '/references/navigation-page/navigation-page--wireframe.png' | path }}" />
+{% include "@headers" %}
+
+{% embed "@sections--plain" with {'classes': 'ecl-pv-l ffa-bg-accent'} %}
+  {% block body %}
+    <div class="section__body">
+      <h2 class="ecl-h2 ecl-mb-l ecl-pa-none ffa-text-center ffa-text-white">
+        What we do by
+        <select name="topics">
```
Comment (emeryro) [inline framework/templates/navigation-page/navigation-page.twig]: We may use diffent text color here, to highlight select list.
You can use ffa color utilities.

File: framework/templates/navigation-page/navigation-page.twig (hunk 4 above)
Comment (kalinchernev) [inline framework/templates/navigation-page/navigation-page.twig]: Added and updated the list to be a bit more realistic.

**Comment by kalinchernev** [comment]
@emeryro thanks for the review! Updates applied.

**Comment by emeryro** [comment]
@kalinchernev you can already add class "ffa-text-shadow" to white headings.
It won't change anything for now, but that way we won't have to update template later.

**Pull request merged**